docs/_articles/en/user-guide/source-diff.md

```diff
@@ -3,33 +3,12 @@ title: Source Diff
 lang: en
 ---
 
-The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between your local project and the metadata in your org.
-
-> NOTICE: The Source Diff feature is currently in beta. If you find any bugs or have feedback, [open a GitHub issue](./en/bugs-and-feedback).
-
-## Setup
-
-Because the Source Diff feature is in beta, you must install a Salesforce CLI plugin. From the terminal, run `sfdx plugins:install @salesforce/sfdx-diff`.
-After the installation is complete, `@salesforce/sfdx-diff` appears in the list of installed plugins when running `sfdx plugins`.
+The Source Diff command allows you to diff metadata types against your default org. This feature makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at Diffs for an individual file or directory.
```

Comment:
@sbudhirajadoc - is there some reason you're capitalizing "Diffs" here but nowhere else that I can see? I would go with l/c.

Reply:
It was capitalized in the note about the feature being in beta. I just left it as is, figuring it had been decided to keep it that way, @ekapner.

Reply:
Ah, are you talking about the second mention? Lemme fix that. Thanks!